docs/content/docs/build/containers.md

```diff
@@ -107,72 +114,48 @@ const { container, services } =
     await client.getContainer(/*container id*/, schema);
 ```
 
-### Deleting a container
-
-Deleting a container is a service-specific feature, so you should refer to the documentation associated with the specific Fluid service you are using. See [Available Fluid Services]({{< relref "service-options.md" >}}) for more information about Fluid service options.
-
-## Container lifecycle and manipulation
-
-### attached/detached
```

Comment:
Entirely removed attached/detached section as it seems as repetition of earlier content in "creating" and "attaching" the container.